src/scenes/home/home.js

```diff
@@ -159,6 +160,10 @@ class Home extends Component {
               component={About}
             />
             <Route
+              path="/families"
```

Comment:
Should we make this `/about/families`?

Reply:
I think it should be it's own separate route because people will usually click it from the home page once we enable a link on the "Military Families and Spouses" card on the landing page.